Tasks/ApiCompat/readme.md

```diff
@@ -0,0 +1,164 @@
+# ApiCompat task
+
+API Compat tool compares two sets of assemblies and determines if the implementation has incompatibilities with the contract.
+
+## How to configure it for BotBuilder Build process
+
+### Configuration of environment variables
+
+- **ApiCompatLibrary:** Comma delimited list of assemblies to compare with ApiCompat. For BotBuilder we will use the following assemblies:
+
+  ```
+  Microsoft.Bot.Builder,Microsoft.Bot.Builder.AI.Luis,Microsoft.Bot.Builder.AI.QnA,Microsoft.Bot.Builder.ApplicationInsights,Microsoft.Bot.Builder.Azure,Microsoft.Bot.Builder.Dialogs,Microsoft.Bot.Builder.TemplateManager,Microsoft.Bot.Configuration,Microsoft.Bot.Connector,Microsoft.Bot.Schema.
```

Comment:
I would suggest making it more generic instead of using botbuilder-dotnet as an example